db/schema.rb

```diff
@@ -57,6 +57,7 @@
   add_index "orders", ["user_id"], name: "index_orders_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
+    t.string   "name"
```

Comment:
SO there should not have been an addition - when i did the merge from the latest dev branch into this branch, there was a merge conflict in this file of an empty line. i had to delete the HEAD and merge conflict notes in the schema file around the empty line to get rid of it.